### PR TITLE
Update AMD Simnow article

### PR DESCRIPTION
**Pull request by eddrda**

### CODE REVIEW

File: content/guides/virtualizing/simnow.md [hunk 1]
```diff
@@ -5,42 +5,104 @@ date = "2016-12-23T06:09:57.000Z"
...
+*   [Downloading AMD SimNow](#part_download-s)
+*   [Installing AMD SimNow](#part_installing-s)
+*   [Preparing the Virtual Machine](#part_preparing-vm)
+*   [Running the Virtual Machine](#part_running-vm)
 
-### Downloading Haiku
+
+### <a name="part_download-h"></a> Downloading Haiku 
 
 There are several methods offered to install Haiku. For AMD SimNow will need to download either the iso image or the anyboot image [here](http://www.haiku-os.org/get-haiku) which is a mix of the iso image and the RAW image and can directly be used as an HDD, i.e. it does not require installation.
```
Comment (humdingerb) [inline content/guides/virtualizing/simnow.md]: There are only Anyboot images and RAW images for download. So you can remove the "either the iso image or" and change to "which is a mix of an ISO image and a RAW image" later in the sentence.

File: content/guides/virtualizing/simnow.md [hunk 2]
```diff
@@ -5,42 +5,104 @@ date = "2016-12-23T06:09:57.000Z"
...
+
+Browse to the hard disk image that you just created in the previous part.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image16.png)
+
+Now, click **File** > **Set IDE Secondary Master Image**.
+
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image17.png)
+
+Change File type to _"ISO Images (*.iso)"_ and browse to the downloaded Haiku iso image.
```
Comment (humdingerb) [inline content/guides/virtualizing/simnow.md]: Remove this screenshot and text for the ISO image. There's only Anyboot now.

File: content/guides/virtualizing/simnow.md [hunk 3]
```diff
@@ -5,42 +5,104 @@ date = "2016-12-23T06:09:57.000Z"
...
 
 ##### Using the Anyboot image
 
-You can also use the anyboot image to directly run Haiku without installation in the VM. ![](/files/images/simnow_image20.png) Just choose the anyboot image as your IDE Primary Master, and you're ready to go!
+![](https://sites.google.com/site/nyhusr/Home/haiku-os-files/haikuarticles/simnow/image20.png)
+
+You can also use the anyboot image to directly run Haiku without installation in the VM. Just choose the anyboot image as your IDE Primary Master, and you're ready to go!
```
Comment (humdingerb) [inline content/guides/virtualizing/simnow.md]: Remove the "also". The screenshot shows the stone old alpha and the filename has a .image suffix, probably forcing you to filter for *.* (any file).
Today's Haiku images are named "haiku-nightly-anyboot.iso". While having the "nightly" in there isn't totally future-proof (once we have another release), but beats the old alpha naming. Just rename the image for the screenshot.

File: content/guides/virtualizing/simnow.md (hunk 1 above)
Comment (eddrda) [reply content/guides/virtualizing/simnow.md]: Should I also change the link to https://download.haiku-os.org/, which contains nightly builds?

File: content/guides/virtualizing/simnow.md (hunk 3 above)
Comment (eddrda) [reply content/guides/virtualizing/simnow.md]: I don't know if I can recreate this screenshot. Should we add a note or I use photo editing software?

Comment (eddrda) [reply content/guides/virtualizing/simnow.md]: I think I'd better follow this guide and replace all the old screenshots.

File: content/guides/virtualizing/simnow.md (hunk 1 above)
Comment (humdingerb) [reply content/guides/virtualizing/simnow.md]: No, after the next release "get-haiku" is the right place to go. And the nightlies are linked from there until then, too.

File: content/guides/virtualizing/simnow.md (hunk 3 above)
Comment (humdingerb) [reply content/guides/virtualizing/simnow.md]: I agree. Thanks!

**Comment by humdingerb** [comment]
Looks very good. One thing though: Do add all the screenshots to the PR and put them into static/files/guides/virtualizing/simnow/ . Adjust the links to e.g. "/files/guides/virtualizing/simnow/simnow1.png".

**Comment by humdingerb** [comment]
Thanks!

**Comment by diversys** [comment]
Imaged are missing from https://www.haiku-os.org/guides/virtualizing/simnow

**Comment by humdingerb** [comment]
Fixed. I overlooked that, because it showed up in the PR preview.
All @eddrda's fault, of course, because he didn't use the URL I provided as example... :) :D

**Pull request merged**